typescript/schemas/src/internal/schemas.ts

```diff
@@ -1,5 +1,45 @@
 import { FoxgloveEnumSchema, FoxgloveMessageSchema } from "./types";
 
+const Duration: FoxgloveMessageSchema = {
+  type: "message",
+  name: "Duration",
+  description: "A duration of time, composed of seconds and nanoseconds",
+  rosEquivalent: "std_msgs/Duration",
+  protoEquivalent: "google.protobuf.Duration",
+  fields: [
+    {
+      name: "sec",
+      type: { type: "primitive", name: "int32" },
+      description: "The number of seconds in the duration",
+    },
+    {
+      name: "nsec",
+      type: { type: "primitive", name: "uint32" },
+      description: "The number of nanoseconds in the positive direction",
+    },
+  ],
+};
+
+const Time: FoxgloveMessageSchema = {
+  type: "message",
+  name: "Time",
+  description: "A timestamp composed of seconds and nanoseconds",
+  rosEquivalent: "std_msgs/Time",
```

Comment:
How do we support ROS2 equivalent? which is `builtin_interfaces/Time` ?

Reply:
Good question.

https://github.com/foxglove/foxglove-sdk/blob/aa7ab0727e054900642ffe2cc218bf684f3a0a53/typescript/schemas/src/internal/generateRos.ts#L32

Rather than duplicating those hardcoded values in our documentation too, I think I'll add a `ros2Equivalent`.